src/screens/KeyManagement/NonEVMKeys.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react'
+
+import Grid from '@material-ui/core/Grid'
+import CircularProgress from '@material-ui/core/CircularProgress'
+
+import { NonEVMKeysCard } from './NonEVMKeysCard'
+import { useNonEvmAccountsQuery } from 'src/hooks/queries/useNonEvmAccountsQuery'
+
+const SCHEMAS = {
+  aptosKeys: {
+    title: 'Aptos',
+    fields: [
+      { label: 'Public Key', key: 'id', copy: true },
+      { label: 'Account', key: 'account', copy: true },
+    ],
+  },
+  solanaKeys: {
+    title: 'Solana',
+    fields: [{ label: 'Public Key', key: 'id', copy: true }],
```

Comment:
since each key always have `{ label: 'Public Key', key: 'id', copy: true }`, , i wonder if we have always in the table, so we dont have to specific this for every new chain key?